File: Clg_Placement/Java/_12.SmallerAndLarger.java
class Solve {
    int[] getMoreAndLess(int[] arr, int n, int x) {
        int[] result = new int[2];
        result[0] = 0;
        result[1] = 0;
        for(int i=0;i<n;i++){
            if(arr[i]==x){
                result[0]++; 
                result[1]++;
            }
            else if(arr[i]<x) result[0]++;
            else result[1]++;
        }
        return result;
    }
}

File: Clg_Placement/C++/Arrays/_12.SmallerAndLarger.cpp
class Solution{
public:	
	vector<int> getMoreAndLess(int arr[], int n, int x) {
	    int less=0,more=0;
	    for(int i=0;i<n;i++){
	        if(arr[i]==x) less++,more++;
	        else if(arr[i]<x)less++;
	        else more++;
	    }
	    return {less,more};
	}
};
